emiluz/README.md

```diff
@@ -0,0 +1,148 @@
+
+You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_30% of total time_)
+Predict the output of each of the following expressions, and explain the rules which determine that answer.  (_Hint: The rules for operator `==` are complicated, but do your best!_)
+
+If the output depends on the value or type of variable _x_, identify the conditions (what _x_ is) when the expression will output true (or false, if that's simpler).  Assume the cases are independent, and _x_ is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+a) "1" == 1 true. One is equal to One.
+
+b) "1" === 1 false. String One is not strict equal to One.
+
+c) x == 'x' The X is not defined therefore X is not equal to string.
+
+d) x == (x+'') The X is not defined.
+
+e)'' == ' ' The answer is false, null
+
+f) x = true
+
+g) var x; x == 'undefined'` false. X is not equal to undefined
+
+h) '9'<'10' false. nine is not greater than 10
+
+i) typeof x + 1 === "number" false. undefined1 is not strict equal to number
+
+j)typeof x % 2 === "number" false. NaN is not strict equal to number
+
+k) typeof (x % 2) === "number" false. X is undefined therefore is not strict equal to number
+
+l) x++ == ++x The X is undefined therefore is not equal to.
+
+m) ++x == x++ The X is undefined therefore is not equal to.
+
+n) "1"+x == 1+x The X is undefined therefore is not equal to.
+
+o) "0"+1 == 1 true. One is equal to One
+
+p) (typeof (x+1))===(typeof x)	The X is not defined therefore is not strict equal.
+
+q) (x*1 == x) || ((typeof x) != "number") The X is undefined
+
+r) (x=(typeof (x+(typeof x))))==x The X is underfined
+
+---
+
+_All of the following can be solved with ordinary expressions and global variables with primitive values.  You don't need functions, loops, or other topics beyond our first two classes._
+
+ 2) (_10%_)
+
+Assume variables x, y, and z are numbers.
+
+a) Write an expression for the mean (i.e. average) of x, y, and z.
+
+	var a= (x+y+z)/3;
+
+b) Write a series of expressions to adjust each of x, y, and z halfway toward the mean of the three.
+That is, reset the value of each variable to something new based on its previous value.
+
+	x=x+(a-x)/2
+	y=y+(a-y)/2
+	z=z+(a-z)/2
+
+
+3) (_20%_)
+
+Suppose you're encoding geometric shapes in a Cartesian (2D) coordinate system, and you represent a rectangle with four numeric variables:
+
+- _l_ : horizontal position of left edge (relative to some origin);
+- _r_ : horizontal position of right edge;
+- _t_ : vertical position of top edge;
+- _b_ : vertical position of bottom edge.
+
+a) Write an expression for the rectangle's area.
+
+	a= (l + r) * (t + r)
+
```

Comment:
For this one the area should be a = (r-l)*(t-b). The variables are relative positions so by subtracting one from the other you get the distance between them.